omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,424 @@
+    $ bin/omero obj new Project name=Project-of-user
+    # Link the containers
+    $ bin/omero obj new ProjectDatasetLink parent=Project:17 child=Dataset:13
+    # Import into created Dataset
+    $ bin/omero import ~/Desktop/CMPO1.png -T Dataset:name:Dataset-of-user
+
+.. _Workflow 3:
+
+Workflow 3: Analyst
+-------------------
+Typically, the Analyst role in OMERO is to
+
+- read the data (always possible, see :ref:`Workflow 1`: Data viewer)
+- change and save the rendering settings of the images (enabled by
+  :term:`Write Data` privilege, exception is
+  Private groups, where they cannot save rendering settings)
+- annotate the data (enabled by :term:`Write Data` privilege, but not
+  possible in Private groups)
+- draw and save ROIs on other users' images (enabled by :term:`Write Data`
+  privilege, but no saving in Private groups possible)
+- upload and attach result files to the analyzed images (enabled by
+  :term:`Write Data` privilege, except Private groups, where attaching
+  is not possible)
+- create Projects and Datasets for newly imported images in groups they
+  are not a member of (enabled by :term:`Write Data` privilege)
+- import new images resulting from image analysis into their own group, then
+  moving the images into the group of the original data owner (enabled by
+  :term:`Chgrp` privilege) into these Projects and Datasets or linking
+  the images to existing Projects and Datasets of the original data owner
+  (enabled by :term:`Write Data` privilege)
+- (possibly) changing the ownership of the newly created conainers and
+  contained result images to the users (enabled by :term:`Chown` privilege)
+- upload, edit and delete official scripts usable by all OMERO users
+  (enabled by :term:`Upload Scripts` privilege)
+
+Client details:
+
+- OMERO.insight or Insight-ImageJ plugin: Analyst has to be a member of the
+  group where the data is located. They can draw ROIs and extract analysis
+  results from the ROIs and data in any type of group. They can save ROIs
+  except in Private groups. They can upload official scripts in
+  OMERO.insight (any group type, Analyst does not have to be a member of
+  any particular group for script upload in OMERO.insight).
+
+- OMERO.web, OMERO.insight, Insight-ImageJ plugin: Analyst can adjust
+  rendering settings and save them, upload
+  attachments with results and annotate (for example tag, key-value pairs,
+  rating, commenting). These actions are not permitted in Private groups
+  with images belonging to others.
+  See :help:`rendering <managing-data#rendering>`,
+  :help:`annotating <managing-data#annotating>`,
+  :help:`attaching files <managing-data#attach>`,
+  :help:`attaching data <managing-data#attach>`.
+
+- CLI: Upload of official scripts is allowed (in any group type,
+  see :doc:`/developers/scripts/user-guide` and below).
+  Upload of attachments with results, annotating
+  (not in private group), creating containers, import of resulting images
```

Comment:
Possible, checked. Care has to be taken in private groups (without chown the owner of the original data will not see the images which Analyst (light admin) imported for them, because these images belong to Analyst.
Clarification added to doc.